docs/supportedchains/supportedTokens.md

```diff
@@ -5,9 +5,9 @@ sidebar_position: 2
 
 # ERC20 Gas Payments
 
-Our Paymaster not only allows you to sponsor gasless tokens for your users but also allows for users to pay for tokens in supported ERC20 tokens. Below are supported tokens grouped by chain.
+Our Paymaster not only allows you to sponsor gasless tokens for your users but also allows users to pay for tokens in supported ERC20 tokens. Below are supported tokens grouped by chain.
```

Comment:
"Our Paymaster not only allows you to sponsor gasless tokens for your users"

it should be

"Our Paymaster not only allows you to sponsor transactions for your users"...